src/math/curve-iterator.js

```diff
@@ -0,0 +1,225 @@
+Object.assign(pc, (function () {
+    'use strict';
+
+    /**
+     *
+     * @constructor
+     * @name pc.CurveIterator
+     * @classdesc CurveIterator performs fast evaluation of a curve by caching
+     * knot information where possible.
+     * @description Creates a new curve iterator.
+     * @param {Curve} curve The curve to iterator over
+     * @param {Number} time The time to start iteration, defaults to 0.
+     */
+    var CurveIterator = function (curve, time) {
+        this.curve = curve;
```

Comment:
Coding standard : private variables should start with _

Reply:
done.